.changeset/hip-owls-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+preserve user's readme on project creation (#1942)
```

Comment:
```suggestion
Preserve any existing user's README when initializing a project (#1942)
```

What about this wording? I think something like this will look better in our release notes.